hw/ip/tlul/generic_dv/xbar_sim_cfg.hjson

```diff
@@ -17,14 +17,31 @@
   // Fusesoc core file used for building the file list.
   fusesoc_core: "lowrisc:dv:{dut}_sim:0.1"
 
-  // Testplan hjson file: set in the autogenerated sim cfg file that imports this.
-  // testplan: ""
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/top_earlgrey/ip/{dut}/data/autogen/{dut}_testplan.hjson"
 
   // Set the path to testplan md file as it's not in the default location.
   testplan_doc_path: "hw/ip/tlul/doc/dv_plan/#testplan"
 
-  // Add xbar specific exclusion files.
-  vcs_cov_excl_files: ["{tool_srcs_dir}/xbar_cov_excl.el"]
+  // Add xbar specific exclusion and cover cfg files.
+  overrides: [
+    {
+      name: vcs_cov_excl_files
+      value: ["{tool_srcs_dir}/xbar_cov_excl.el"]
+    }
+    {
+      name: default_vcs_cov_cfg_file
+      value: "-cm_hier {tool_srcs_dir}/xbar_cover.cfg"
+    }
+  ]
+
+  // Bypass VCS CHECK_SUM check as the exclusion file is generated without proper CHECK_SUM value
+  cov_analyze_opts: ["-excl_bypass_checks"]
+  cov_report_opts:  ["-excl_bypass_checks"]
```

Comment:
Hmm, these are VCS specific opts. Probably do something like this:
```suggestion
  vcs_cov_analyze_opts: ["-excl_bypass_checks"]
  vcs_cov_report_opts:  ["-excl_bypass_checks"]
  
  cov_analyze_opts: ["{tool}_cov_analyze_opts"]
  cov_report_opts: ["{tool}_cov_report_opts"]  
```

Need to find a better way to do things like this....

Reply:
Updated, thanks